bridge/opentracing/bridge.go

```diff
@@ -158,7 +158,13 @@ func (s *bridgeSpan) SetOperationName(operationName string) ot.Span {
 func (s *bridgeSpan) SetTag(key string, value interface{}) ot.Span {
 	switch key {
 	case string(otext.SpanKind):
-		// TODO: Should we ignore it?
+		type readSpanKindSpan interface {
+			SpanKind() trace.SpanKind
+		}
+		// only add span kind tag when otel span kind is not set (internal)
```

Comment:
`SpanKindUnspecified` is the "unset" zero-value of a span kind.

Reply:
Thanks @MrAlias!

I included `SpanKindUnspecified` in the if condition, and updated the comment. But I have to keep the `SpanKindInternal` in the condition because the bridge enforces the default span kind to be `SpanKindInternal` even without any span kind explicitly specified when starting the span.
https://github.com/open-telemetry/opentelemetry-go/blob/eb2b89f3356cdf162b9aeaff39210a4f3d49f06c/bridge/opentracing/bridge.go#L502-L503